core/Pos/Crypto/Signing.hs

```diff
@@ -125,10 +125,10 @@ shortPublicKeyHexF :: Format r (PublicKey -> r)
 shortPublicKeyHexF = fitLeft 8 %. fullPublicKeyHexF
 
 -- | Parse 'PublicKey' from base64 encoded string.
-parseFullPublicKey :: (Bi PublicKey) => Text -> Maybe PublicKey
+parseFullPublicKey :: Text -> Maybe PublicKey
 parseFullPublicKey s = do
     b <- rightToMaybe $ Base64.decode s
-    rightToMaybe $ Bi.decodeFull b
+    PublicKey <$> rightToMaybe (CC.xpub b)
```

Comment:
See [here](https://github.com/adinapoli-iohk/cardano-sl/commit/88849efe8609d4d20b5e9f76e30a2f60c8d972cd) for the full story around this change.